src/Symfony/Component/Notifier/Bridge/Mobyt/MobytTransport.php

```diff
@@ -33,11 +33,15 @@ final class MobytTransport extends AbstractTransport
     private $from;
     private $typeQuality;
 
-    public function __construct(string $accountSid, string $authToken, string $from, string $typeQuality, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    public function __construct(string $accountSid, string $authToken, string $from, string $typeQuality = null, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

Comment:
Why do we want to make that parameter nullable anyway?

```suggestion
    public function __construct(string $accountSid, string $authToken, string $from, string $typeQuality = MobytOptions::MESSAGE_TYPE_QUALITY_LOW, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

Reply:
I would suggest that staying BC from the ctor signature is easier if the default value can be handled in the ctor itself, isn't it? 🤔

Reply:
Ah and we need to know the default value in the transport factory too:
https://github.com/symfony/symfony/blob/73509d974b9da4cd84eb10daa4c2a0b0181e3ff6/src/Symfony/Component/Notifier/Bridge/Mobyt/MobytTransportFactory.php#L27-L43